pkg/util/config/feature_gate.go

```diff
@@ -37,6 +37,8 @@ const (
 	// values override the default set by allAlpha.
 	// e.g. allAlpha=false,newFeature=true will result in newFeature=true
 	allAlpha = "allAlpha"
+
+	externalTrafficLocalOnly = "externalTrafficLocalOnly"
```

Comment:
wrt naming, an ideal convention would express "this gate controls the
availability of an optional feature" rather than "this gate controls the
enablement of a new behavior".  I'm not SURE we need this, but it nags at
me.  E.g. if we (eventually) wanted to flip this new behavior to be
default, what we call THAT gate?

On Mon, Aug 22, 2016 at 9:39 AM, Jeff Lowdermilk <notifications@github.com>
wrote:

> In pkg/util/config/feature_gate.go
> <https://github.com/kubernetes/kubernetes/pull/29409#discussion_r75712417>
> :
>
> > @@ -37,6 +37,8 @@ const (
> >  	// values override the default set by allAlpha.
> >  	// e.g. allAlpha=false,newFeature=true will result in newFeature=true
> >  	allAlpha = "allAlpha"
> > +
> > +	externalTrafficLocalOnly = "externalTrafficLocalOnly"
>
> "LocalOnlyLBTraffic"?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/29409/files/59883dc63dbea4bc31ef2ef85d3274a0ea383e78..9875b3b6f8892cf2c1e4c0aa2009b5161448c094#r75712417>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AFVgVD6wJRcQI57dkZ88jIa2yfQk7RHWks5qidDagaJpZM4JSUXH>
> .
>